src/pull_automerge/core.clj

```diff
@@ -94,27 +94,37 @@
   (println "Get statuses result:" (result :status))
   result)
 
+(defn filter-by-label [ response-data label ]
+  (filter 
+    #(> (count (filter (fn[m] (= (m :name) label)) (% :labels))) 0) 
```

Comment:
I'm considering making this into its own function rather than havining an anonymous function here... maybe something like
```
(defn contains-label [ labels label ]
  (> (count (filter #(= (% :name) label)) labels)) 0))
```

(I haven't verified that this actually works, but I think it might help make it more readable. What do you think?)